src/intTest/kotlin/no/nav/personbruker/dittnav/eventaggregator/common/kafka/ConsumerTestIT.kt

```diff
@@ -0,0 +1,131 @@
+package no.nav.personbruker.dittnav.eventaggregator.common.kafka
+
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import no.nav.brukernotifikasjon.schemas.Beskjed
+import no.nav.brukernotifikasjon.schemas.Nokkel
+import no.nav.personbruker.dittnav.eventaggregator.beskjed.AvroBeskjedObjectMother
+import no.nav.personbruker.dittnav.eventaggregator.common.SimpleEventCounterService
+import no.nav.personbruker.dittnav.eventaggregator.common.ThrowingEventCounterService
+import no.nav.personbruker.dittnav.eventaggregator.common.database.kafka.util.KafkaTestUtil
+import no.nav.personbruker.dittnav.eventaggregator.common.exceptions.RetriableDatabaseException
+import no.nav.personbruker.dittnav.eventaggregator.common.exceptions.UnretriableDatabaseException
+import no.nav.personbruker.dittnav.eventaggregator.config.EventType
+import no.nav.personbruker.dittnav.eventaggregator.config.Kafka
+import no.nav.personbruker.dittnav.eventaggregator.nokkel.createNokkel
+import org.amshove.kluent.`should be equal to`
+import org.amshove.kluent.`should be greater than`
+import org.amshove.kluent.`should contain same`
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.junit.jupiter.api.Test
+
+class ConsumerTestIT {
+
+    private val beskjedEvents = (1..10).map { createNokkel(it) to AvroBeskjedObjectMother.createBeskjed(it) }.toMap()
+
+
+    @Test
+    fun `Should attempt process each event exactly once if no exceptions are thrown`() {
+
+        val topic = "kafkaConsumerStateTestTopic"
+        val embeddedEnv = KafkaTestUtil.createDefaultKafkaEmbeddedInstance(listOf(topic))
```

Comment:
Dette ser bra ut. Men kunne vi her prøvd å brukt fire partisjoner, i stede for to som er default?